stanford/playbooks/coursegraph.yml

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: coursegraph
+  when: redeploy | default(False) | bool
+- name: "Setup {{ COMMON_HOSTNAME }}"
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
+  gather_facts: True
+  become: true
+  vars:
+    CLUSTER_NAME: coursegraph
+  vars_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
+  roles:
+    - neo4j
```

Comment:
Shouldn't the neo4j role be modified to include Tony's password change work?

Reply:
Since we're already locking the machine down via SSH and managing users that way,
we don't really need to further lock down that account.

The original idea was that we would create individual users in the neo4j web-ui for each real-world user (hopefully via oauth or some form of SSO). Given that, we would have wanted to lock down the default `neo4j` user to prevent unauthorized access.

But now that we're not relying on the web-ui for personal authentication, but rather enforcing that via SSH access, we don't really need to lock this default account down. In fact, we may as well keep it easy with the default so it's one less password for everyone to manage and keep in sync.

For reference, the reason no such task exists in the role yet is because edx doesn't bother setting a new password either. They similarly lock things down via network-level access, though in their case, they restrict access to w/in the EdX VPN.